> Am I doing something wrong? The command seems to produce no output on my machine.
> 
> ```
> $ cd fuzz
> $ make build-target
> cargo +nightly rustc --bin   -- \
> 	-C debuginfo=full \
> 	-C debug-assertions \
> 	-C passes='sancov-module' \
> 	-C llvm-args='-sanitizer-coverage-level=3' \
> 	-C llvm-args='-sanitizer-coverage-inline-8bit-counters' \
> 	-Z sanitizer=address
> error: "--bin" takes one argument.
> Available binaries:
>     accounts_call
>     accounts_call_random
>     accounts_parse_call_message
>     accounts_parse_call_message_random
>     address_bech_32_parse_serde
>     address_bech_32_try_from_bytes
>     bank_call
>     bank_parse_call_message
>     parse_address
>     share_deserialize
> 
> make: *** [Makefile:14: build-target] Error 101
> 
> # Trying to run awk separately
> $ awk -F' = ' '/^\[\[bin\]\]/,/^$$/{ if ($$1 == "name") { gsub(/"/, "", $$2); print $$2 } }' Cargo.toml
> $ # No output!
> ```

Thank you for raising that. I was hoping that CI will catch any errors on linux, but it looks like I have to do it myself.

--- a/fuzz/Makefile
+++ b/fuzz/Makefile
@@ -1,6 +1,6 @@
 .PHONY: help
 
-BINARIES := $(shell awk -F' = ' '/^\[\[bin\]\]/,/^$$/{ if ($$1 == "name") { gsub(/"/, "", $$2); print $$2 } }' Cargo.toml)
+BINARIES := $(shell awk -F'=[[:space:]]*"|"[[:space:]]*$$' '/^\[\[bin\]\]/,/^$$/{ if ($$1 ~ /name[[:space:]]*/) { print $$2 } }' Cargo.toml)
 PROFILE ?= debug
 ARGS ?=
 

--- a/fuzz/Makefile
+++ b/fuzz/Makefile
@@ -10,6 +10,7 @@ help: ## Display this help message
 check: ## Checks that fuzz member compiles
 	cargo check
 
+# Needs nightly for `-Z sanitizer=address`
 build-target: ## Build the target fuzz
 	cargo +nightly rustc --bin $(TARGET) $(ARGS) -- \
 		-C debuginfo=full \

